app/scss/midnight-theme.scss

```diff
@@ -0,0 +1,365 @@
+/**
```

Comment:
How comes this duplicates `themes/theme.scss` instead of also using it as the template in the manner of `palm-theme.scss`?

Reply:
Good idea! I will do it after the LEAF theme if that's okay with you